src/model/ImageCollection.hpp

```diff
@@ -48,18 +49,24 @@ class ImageCollection : public QObject
 
   ImageDocument * current() const { return items_.at(currentIndex_); }
   int currentIndex() const { return currentIndex_; }
-  
+  QList<int> const& recentItems() const { return recentItems_; };
+
   void setCurrentIndex(int i);
 
   Collection const& vector() const { return items_; }
 
 signals:
   void itemsChanged();
+  void recentItemsChanged();
```

Comment:
No. I added it without much thinking just for consistency and because I was not sure whether or not QML needs it. Seems to work without it, I will remove it.